core/templates/dev/head/pages/learner_dashboard/LearnerDashboard.js

```diff
@@ -137,6 +137,13 @@ oppia.controller('LearnerDashboard', [
       return ($window.innerWidth < 500);
     };
 
+    $scope.getVisibleExplorationList = function(startCompletedExpIndex) {
```

Comment:
Should probably have something in the varname that alludes to "completed explorations", since multiple types of exploration tiles are being shown on the learner dashboard.